content/blog/highlights-of-2023/index.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: blog-post
+title: Highlights of 2023 
+description: "A Year of Growth: Highlights from 2023 for Haystack and our Community"
+featured_image: thumbnail.png
+images: ["blog/highlights-of-2023/thumbnail.png"]
+featured_image_caption: Highlights of 2023
+alt_image: Highlights of 2023
+toc: True
+date: 2024-01-05
+last_updated:  2024-01-05
+authors:
+  - Bilge Yucel
+tags: ["Open Source", "Haystack 2.0", "Community"]
+---	
+
+> *This post was originally shared through the Haystack newsletter. [Subscribe now](https://landing.deepset.ai/haystack-community-updates) to stay in the loop on all things Haystack, including the latest updates, new features, captivating content, and upcoming exciting events!* 🗞️
+
+What a fantastic year it has been! In 2023, the world of AI saw tremendous progress, making it an exciting time for learning and growth. As we wind down and approach the new year, it's a good moment to take a breather, look back on the past year, and appreciate the highlights before gearing up for what's ahead. We've taken some time to review the noteworthy moments that made 2023 special for Haystack and its community.
+
+A big thank you for being part of this journey! Here's to a wonderful New Year filled with dizzying generative AI and NLP advancements 🥂✨
+
+## 💫 Haystack Highlights
+
+- The biggest highlight of the year was [**Haystack 2.0-Beta**](https://haystack.deepset.ai/blog/introducing-haystack-2-beta-and-advent), which is a more flexible, extendable, customizable version of Haystack v1 💯
```

Comment:
nit: it feels a little awkwardly worded to call Haystack 2.0 a version of Haystack v1 even though I suppose it's technically true. You could drop the V1 from this sentence and it would read slightly more smoothly. Just a suggestion though.